### PR TITLE
fix typo and misleading statement (again)

### PR DESCRIPTION
**Pull request by earloc**
#5

### CODE REVIEW

**Comment by lbargaoanu** [comment]
I didn't see this one and now there are conflicts. But the text here is hard to understand. You might want to try again.

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ AutoMapper.Extensions.EnumMapping
 
 ### Summary
 
-The AutoMapper.Extensions.EnumMapping library gives you control about your enum values mappings. Normally enums are automatically mapped bij AutoMapper as an integer, but you have no control about custom mappings. It is possible to create a custom type converter for every enum.
+The AutoMapper.Extensions.EnumMapping library gives you control about your enum values mappings. Normally, enums are automatically mapped by AutoMapper by performing a case-insensitive TryParse, before falling back to mapping by using the internal-integer value, but you have no control about custom mappings. It is possible to create a custom type converter for every enum.
```
Comment (lbargaoanu) [inline README.md:10]: Ok, it is `TryParse`, but on what? :) Maybe say smth like map by string values? Then fallback to mapping by the underlying numeric value.

**Comment by lbargaoanu** [comment]
The latest version is [here](https://github.com/AutoMapper/AutoMapper/blob/c7278ef7c3dc7878ed10815f77c9fc6f9ef2e5b2/src/AutoMapper/Mappers/EnumToEnumMapper.cs#L20). But see https://github.com/AutoMapper/AutoMapper/issues/3562.

**Comment by earloc** [comment]
hmm, maybe it would be better to not make any assumptions about AM´s default behavior in this doc here at all?

**Comment by lbargaoanu** [comment]
That was my idea :) I simply removed that line.

**Pull request closed without merging**